AjaxFetch/css/styles.css

```diff
@@ -0,0 +1,19 @@
+.well{
+  min-height: 20px;
+  padding: 19px;
+  margin: 0 auto 20px;
+  background-color: #f5f5f5;
+  border: 1px solid #e3e3e3;
+  border-radius: 4px;
+  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.05);
```

Comment:
You also can setup https://www.npmjs.com/package/gulp-autoprefixer, and don't write this param in your style.